pkg/grpc/authz/or/or.go

```diff
@@ -35,14 +35,11 @@ func Or(authorizers ...authz.Authorizer) authz.Authorizer {
 // SensorOr returns an Authorizer that authorizes any sensor,
 // or anything that the passed authorizer authorizes.
 func SensorOr(authorizer authz.Authorizer) authz.Authorizer {
-	return Or(
-		idcheck.SensorsOnly(),
-		authorizer,
-	)
+	return Or(idcheck.SensorsOnly(), authorizer)
```

Comment:
nit: unexpected change

Reply:
Indeed. When exploring the `ServiceOr` option, I found the coding style was inconsistent between `SensorOr` and `ScannerOr`. I'll revert this piece of the change. Style consistency may be brought in a follow-up PR.